pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -748,6 +752,21 @@ func (r *Builder) PopulatorVolumes(vmRef ref.Ref, annotations map[string]string,
 	return
 }
 
+func (r *Builder) mapStorageDomainToStorageClass() (map[string]string, error) {
+	sdToStorageClass := make(map[string]string)
+	dsMapIn := r.Context.Map.Storage.Spec.Map
+	for i := range dsMapIn {
+		mapped := &dsMapIn[i]
```

Comment:
I think we can just use mapped in the for statement directly, we don't take the address anyway so there wouldn't be loopvar issues

Reply:
yep, copy-pasted, done